admin/base/views.py

```diff
@@ -74,3 +57,15 @@ def form_valid(self, form):
     @property
     def success_url(self):
         raise NotImplementedError
+
+
+class GuidView(DetailView):
+    def __init__(self):
+        self.guid = None
```

Comment:
Super is called the line below. The extra things for guids and similar have been taken care of in one PR but this and at least 11 others like it are waiting for that one to get merged. I believe #5013. @sloria